apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -114,19 +117,19 @@ init_per_group(contracts, Config) ->
 
     %% Save account information.
     Accounts = #{acc_a => #{pub_key => APubkey,
-			    priv_key => APrivkey,
-			    start_amt => StartAmt},
-		 acc_b => #{pub_key => BPubkey,
-			    priv_key => BPrivkey,
-			    start_amt => StartAmt},
-		 acc_c => #{pub_key => CPubkey,
-			    priv_key => CPrivkey,
-			    start_amt => StartAmt},
-		 acc_d => #{pub_key => DPubkey,
-			    priv_key => DPrivkey,
-			    start_amt => StartAmt}},
+                            priv_key => APrivkey,
+                            start_amt => StartAmt},
+                 acc_b => #{pub_key => BPubkey,
+                            priv_key => BPrivkey,
+                            start_amt => StartAmt},
+                 acc_c => #{pub_key => CPubkey,
+                            priv_key => CPrivkey,
+                            start_amt => StartAmt},
+                 acc_d => #{pub_key => DPubkey,
+                            priv_key => DPrivkey,
+                            start_amt => StartAmt}},
     [{accounts,Accounts},{node_name,NodeName}|Config];
-init_per_group(test_group, Config) ->		%Do nothing
+init_per_group(test_group, Config) ->           %Do nothing
```

Comment:
Unused? (Comment not in scope of this PR.)

Reply:
Same for end_per_group

Reply:
They were originally added when I was doing some testing then just not removed.